src/js/angular/getImgService.js

```diff
@@ -0,0 +1,38 @@
+(function (angular) {
```

Comment:
One small update to the directory name. I like that you have the app within the `src/` directory, however, `angular` is too generic. For smaller apps, I like to group the different constructs together, like so:

```
/src
  /controllers
  /directives
  /services 
  ...
  index.html
```

The directory structure would most likely change again as the app grows and features start to take shape. When that happens, it's common to group files together in a "birds of a feather (flock together)" structure. For example, if you had a navigation AND the Flickr slideshow, you may want to structure like so:

```
/src
  /flickr
    /controllers
    /directives
    ...
  /navigation
    /controllers
    /directives
    ...
```

For now, please just change `src/angular` to `src/services`.